examples/webpack.common.js

```diff
@@ -25,6 +25,6 @@ module.exports = {
     },
     performance: {
         hints: 'error',
-        maxAssetSize: 4.5 * mb,
+        maxAssetSize: 4.7 * mb,
```

Comment:
Because we have more and more code and the binary is bigger and bigger. After Danilo implements his task, this will be handled nicer (without debug symbols).